lib/puma/const.rb

```diff
@@ -124,7 +124,7 @@ module Const
     # we amortize the cost of going back to the reactor for a well behaved
     # but very "greedy" client across 10 requests. This prevents a not
     # well behaved client from monopolizing the thread forever.
-    MAX_FAST_INLINE = 10
+    MAX_FAST_INLINE = ENV.fetch('MAX_FAST_INLINE', 10).to_i
```

Comment:
I think it would be cleaner to make this an attribute of the Server object, which is read from configuration. 

We don't have any env variables setting constants right now, probably shouldn't start.